composer.json

```diff
@@ -1,27 +1,48 @@
 {
-    "name": "bitexpert/composer-phive-bridge",
-    "description": "Update tools via phive on composer update",
-    "type": "composer-plugin",
-    "license": "MIT",
-    "authors": [
-        {
-            "name": "Andreas Heigl",
-            "email": "a.heigl@bitexpert.de"
-        }
+  "name": "bitexpert/composer-phive-bridge",
+  "description": "Update tools via phive on composer update",
+  "type": "composer-plugin",
+  "license": "MIT",
+  "authors": [
+    {
+      "name": "Andreas Heigl",
+      "email": "a.heigl@bitexpert.de"
+    }
+  ],
+  "require": {
+    "php": "^7.4|~8.0.0",
+    "composer-plugin-api": "^2.0"
+  },
+  "require-dev": {
+    "composer/composer": "^2.0",
+    "squizlabs/php_codesniffer": "^3.5",
```

Comment:
Shouldn't we use phive to include these tools?

Reply:
Good catch. I mostly copied the tools from our other repos. Since we want to promote to use PHIVE, I guess we should be good citizens and make use of PHIVE in this project :)